browser/brave_vpn/win/brave_vpn_wireguard_service/service/wireguard_service_runner.cc

```diff
@@ -188,6 +188,11 @@ HRESULT WireguardServiceRunner::InitializeComSecurity() {
     return E_ACCESSDENIED;
   }
 
+  // Don't allow guest account to instantiate.
+  if (!dacl.AddDeniedAce(Sids::Guests(), com_rights_execute_local)) {
```

Comment:
This alone should fix the complaint. This permission will deny the guest account from making the COM call.

Reply:
cc: @thypon

Reply:
Can we allow only the current user to make the request?

Reply:
The above is a bit tricky to read - but it's restricting the COM component to only allow calls from:
- System
- Administrators
- Person who is logged in who is NOT a guest